earn/src/components/portfolio/modal/WithdrawModal.tsx

```diff
@@ -234,12 +246,9 @@ export default function WithdrawModal(props: WithdrawModalProps) {
     };
   }, [refetchMaxWithdraw, refetchMaxRedeem, isOpen]);
 
-  const maxWithdrawBalance = useMemo(() => {
-    if (maxWithdraw) {
-      return new Big(maxWithdraw.toString()).div(10 ** selectedOption.decimals).toString();
-    }
-    return '0.00';
-  }, [maxWithdraw, selectedOption]);
+  const bigWithdrawAmount = inputValue ? new Big(inputValue) : new Big(0);
+  const bigMaxWithdraw: Big = maxWithdraw ? new Big(maxWithdraw.toString()) : new Big(0);
```

Comment:
Big's can handle scientific notation, but BigNumber's cannot. That is why this is okay.